### DISCUSSION TITLE
Binary File Format

### DISCUSSION THREAD

**Discussion by mweisgut** [hyrise-mp-22-23/hyrise#120] [General]
<img width="1293" alt="image" src="https://user-images.githubusercontent.com/38314662/221875654-8009fc53-95d5-4359-a052-df518df8e9f6.png">

Regarding the segment metadata:
- Is `Element Count` the number of elements in the dictionary?
- Wouldn't `Element Count in Attribute Vector` for each segment be redundant information since we store `Row Count` per chunk and each segment has the same number of elements?

**Comment by SanJSp** [comment]
1. Yes
2. Yes, AFAIK.

Here are all versions so far, changes highlighted in red:
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/38314662/221871546-e378272f-a507-4a40-9de8-cfde25a9d341.png">
<img width="1309" alt="image" src="https://user-images.githubusercontent.com/38314662/221875444-e04062cd-eba5-4a36-afa1-238019975ef4.png">
